haas/client/node.py

```diff
@@ -0,0 +1,33 @@
+from haas.client.base import *
+from haas.client.client_errors import *
+
+
+class Node(ClientBase):
+    """ Consists of calls to query and manipulate node related
+    objects and relations """
+
+    def list(self, is_free):
```

Comment:
I checked the cli, client and api code for this call. CLI rejects any other options other than `free` and `all`. But api and client library do not have any such restrictions. I mean in absence of the check done at CLI, the API would respond to any string as if it was keyword `all`. I believe this should be fixed at API level.